apps/voting/app/src/vote-utils.js

```diff
@@ -43,22 +55,31 @@ export function getVoteStatus(vote, pctBase) {
 }
 
 export function getVoteSuccess(vote, pctBase) {
-  const { yea, minAcceptQuorum, nay, supportRequired, votingPower } = vote.data
+  const { minAcceptQuorum, nay, supportRequired, votingPower, yea } = vote.data
 
-  const totalVotes = yea.add(nay)
+  const totalVotes = yea.value().add(nay.value())
```

Comment:
Just realized this probably could equal `-2` if both `yea` and `nay` get passed in as `-1`, but, AFAIK, this can't really happen as they're not being set at `-1` at any moment.